src/datepicker/datepicker.js

```diff
@@ -31,66 +31,173 @@ angular.module('ui.bootstrap.datepicker', ['ui.bootstrap.dateparser', 'ui.bootst
   // Modes chain
   this.modes = ['day', 'month', 'year'];
 
-  // Interpolated configuration attributes
-  angular.forEach(['formatDay', 'formatMonth', 'formatYear', 'formatDayHeader', 'formatDayTitle', 'formatMonthTitle'], function(key) {
-    self[key] = angular.isDefined($attrs[key]) ? $interpolate($attrs[key])($scope.$parent) : datepickerConfig[key];
-  });
+  if ($attrs.datepickerOptions) {
+    var datepickerOptions = $scope.$parent.$eval($attrs.datepickerOptions);
```

Comment:
There is a potential flaw with this - if the user breaks the object reference, this will have the old reference cached.